.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    target-branch: main
+    schedule:
+      interval: "daily"
+      time: "07:00"
+    labels:
+      - "dependencies"
+    open-pull-requests-limit: 0
```

Comment:
is it correct to disable creating PRs?
> # Disable version updates for npm dependencies
    open-pull-requests-limit: 0

from https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit